### PR TITLE
Odoo v9 docker image based on the v8 image with Odoo sources instead of OCB

### PR DESCRIPTION
**Pull request by seb-elico**

### CODE REVIEW

**Comment by ajite** [comment]
LGTM to be improved later.

File: bin/boot [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+echo "Search conf file"
+if [[ -z "$ODOO_CONF" ]];
+then
+    file="/opt/odoo/etc/odoo.conf"
+else
+    file=$ODOO_CONF
+fi
+if [ ! -f "$file" ]; then
+    cp /opt/sources/odoo.conf $file
```
Comment (siyuancn) [inline bin/boot]: Same question as https://github.com/Elico-Corp/odoo-docker/pull/4, do we need to consider the case if cp is fail?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,95 @@
+FROM ubuntu:14.04
+MAINTAINER Elico Corp <info@elico-corp.com>
```
Comment (elicoidal) [inline Dockerfile]: info@ doesnot exist (this is not an issue it is forwarded.
should we keep it like this or us contact@ or create a docker@

Comment (seb-elico) [inline Dockerfile]: Indeed contact@elico-corp.com is an option
Sébastien Maillard On 09/25/2015 06:16 PM, Eric Caudal wrote:

File: bin/boot (hunk 1 above)
Comment (seb-elico) [inline bin/boot]: Same answer ;)
Sébastien Maillard /**/ On 09/25/2015 05:57 PM, Siyuan wrote:

Comment (siyuancn) [inline bin/boot]: ok

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**